x-pack/plugins/ml/server/shared_services/shared_services.ts

```diff
@@ -38,6 +42,8 @@ export type SharedServices = JobServiceProvider &
   ResultsServiceProvider &
   MlAlertingServiceProvider;
 
+export type MlServicesProviders = JobsHealthServiceProvider;
```

Comment:
There a bit of inconsistency with the naming, perhaps this should be called `internalServices` or something like that?
Something that is more obvious when compared to `SharedServices` above.

Reply:
Renamed in https://github.com/elastic/kibana/pull/106084/commits/5f886cb99dafdd60a5d5c336eafd3a50ffaa97c9